pkg/storage/chunk/cache/cache.go

```diff
@@ -87,7 +87,7 @@ func IsRedisSet(cfg Config) bool {
 }
 
 func IsGroupCacheSet(cfg Config) bool {
-	return cfg.EnableGroupCache
+	return cfg.GroupCache != nil
```

Comment:
I don't think it'll ever be `nil`?
https://github.com/grafana/loki/blob/7e76375199702a29dff3c925f49f4dc311ad61a7/pkg/storage/chunk/cache/cache.go#L43